.github/templates/README.md

```diff
@@ -0,0 +1,3 @@
+# %%PRETTY_NAME%%
```

Comment:
Since you didn't update the script, this would probably result in something like %Airdrop Freshdesk Extractor%, because you added %% instead of %. Also this readme should probably also contain some extra sections that we want populated, like what is currently in the main readme. I have a separate PR that fixes that though, so just fix the %% here.